lib/voximplant_api/client.rb

```diff
@@ -13,18 +13,21 @@ def initialize(error)
   end
 
   class Client
+    attr_reader :private_key, :private_key_id, :account_id
 
     def initialize(options)
       @account_id = options[:account_id]
       @api_key = options[:api_key]
+      @private_key = options[:private_key]
```

Comment:
Probably here should be a check that only one auth method is enabled at a time

Reply:
Technically, client can use both kind of auth methods at same time. If so, the decision of is request authenticated or not will be completely on api provider side. We can perform validation for at least one auth method provided during client initialization, WDYT?

Reply:
"at least one" sounds good